### ISSUE TITLE
How to train a VGG data set by using Triplet Loss

### ISSUE THREAD

**Issue by thuoctran** [davidsandberg/facenet#790]
Hi,
I am trying to train a VGG dataset by using the train_tripletloss.py.
I used this parameters:
--optimizer ADAGRAD --learning_rate 0.05 --learning_rate_decay_factor 0.98 --learning_rate_decay_epochs 4 --weight_decay 2e-4 --max_nrof_epochs 500 --people_per_batch 720 --images_per_person 5
but, the program is crashed when it was trying to run an evaluate and save a checkpoint model.
